### PR TITLE
Add v3 migration guide

### PR DESCRIPTION
**Pull request by dpikt**
Final task for v3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,9 +109,9 @@ input :start_date, :date, parse_format: '%Y-%m-%d'
 
 This decanter will look up and apply the corresponding `DestinationDecanter` whenever necessary to transform nested resources.
 
-## Exception handling
+### Exceptions
 
-By default, `Decanter#decant` will raise will raise an exception when unexpected parameters are passed. To override this behavior, you can disable strict mode:
+By default, `Decanter#decant` will raise an exception when unexpected parameters are passed. To override this behavior, you can disable strict mode:
```
Comment (inveterateliterate) [inline README.md]: can people use the `.config` syntax to set this to false globally?

Comment (dpikt) [reply README.md]: Yep see here https://github.com/LaunchPadLab/decanter/pull/89/files#diff-04c6e90faac2675aa89e2176d2eec7d8R204

Comment (dpikt) [reply README.md]: Perhaps that's a little hidden though.

Comment (dpikt) [reply README.md]: Added a link

**Pull request merged**